thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/rootcause/impl/ThirdEyeEventEntity.java

```diff
@@ -0,0 +1,51 @@
+package com.linkedin.thirdeye.rootcause.impl;
+
+import com.linkedin.thirdeye.anomaly.events.EventType;
+import com.linkedin.thirdeye.datalayer.dto.EventDTO;
+import com.linkedin.thirdeye.rootcause.Entity;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ThirdEyeEventEntity extends EventEntity {
```

Comment:
With this added, you should be able to remove HolidayEventEntity

Reply:
Removed